vite.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+    test: {
```

Comment:
use `include` and `exclude` patterns to indicate test files. do not rename the test file to `*.test.ts` which is annoying.